lib/application_configuration.rb

```diff
@@ -7,13 +7,8 @@ module ApplicationConfiguration
   # configured.
   def configured_value(config_path, options = {})
     value = CONFIG.dig(*config_path)
-
-    # This is a peculiar situation because value will return {}
-    # on failed queries. However, it is entirely possible that
-    # a query will yield false (as a configured value), so we cannot
-    # use standard .present? or .blank?. We also cannot use .empty?
-    # because TrueClass and FalseClass do not support it
-    if value != {} then value
+    # dig returns nil when key not found
+    if value.present? then value
```

Comment:
@enragednuke's comment about `false` was correct, though, so in this case we'll do the C idiom:

```ruby
if value then value
```